### DISCUSSION TITLE
How to add tauri to existing sveltekit?

### DISCUSSION THREAD

**Discussion by zer09** [tauri-apps/tauri#4676] [Q&A] [answered]
What is the proper way of adding tauri on sveltekit?

I tried `yarn add -D @tauri-apps/cli @tauri-apps/api` then `yarn init tauri`. but running it with  `yarn tauri dev`, I am stuck with this `Warn Waiting for your frontend dev server to start on http://localhost:5173/...`

**Comment by FabianLars** [comment, marked as answer]
This warning means that your frontend is not running or it's running on a different port than tauri expects (`devPath`).

In the first case, you either need to run it manually in a separate terminal (`yarn start` or `yarn dev` or whatever it's caleld), or add it to `beforeDevCommand` if you want tauri to start it for you. Btw you have to do the same for the build command.

**Comment by AminoffZ** [comment]
For anyone else who might have this problem, I had the devPath missing a "/" after the port. Adding it made it work.
